docs/getting-started.md

```diff
@@ -90,6 +92,23 @@ var howManyRequestsDoIHaveLeft = rateLimit?.Remaining;
 var whenDoesTheLimitReset = rateLimit?.Reset;
 ```
 
+However, if in some cases you need to get the Rate Limit directly from Github, you should call `GitHubClient.Miscellaneous.GetRateLimits()`.
+
+Example usage:
+
+```csharp
+GithubClient client; 
+//Create & initialize the client here
+
+var miscellaneousRateLimit = client.Miscellaneous.GetRateLimits();
+
+var rate = miscellaneousRateLimit.Result.Rate;
```

Comment:
According to the docs the `Rate` property is deprecated so our docs should actually steer people towards the proper field (`.Resource.Core`)

Reply:
Could probably also mention the search API rate limit in the code sample too

Reply:
I replaced the `Rate` property with the `.Resource.Core`. Also, I added a sample for the search API.